endoreg_db/services/video_import.py

```
@@ -3,672 +3,1210 @@
+            "Import and anonymization completed for VideoFile UUID: %s", video.uuid
+        )
+        self.logger.info("Raw video stored in: /data/videos")
+        self.logger.info("Processed video stored in: /data/anonym_videos")
 
-        Returns:
-            Path: The path to the created sensitive file.
-        """
-        video_file = video_instance or self.current_video
-        # Always use the currently stored raw file path from the model to avoid deleting external source assets
-        source_path = None
+    def _create_sensitive_file(
```

Pylint (4.0.2)

[refactor] Too many local variables (17/15)

(R0914)